packages/contracts/tests/js/ZEROIssuanceArithmeticTest.js

```diff
@@ -401,6 +401,142 @@ contract('ZERO community issuance arithmetic tests', async accounts => {
     await assert.equal(latestTotalSOVIssued.toString(), calculateSOVAccrual(initialLastIssuanceTime, latestLastIssuanceTime, zusdDeposits).toString())
   });
 
+  // Error tolerance: 1e-3, i.e. 1/1000th of a token
+  it("should issue the expected SOV based on pre-defined params (1)", async () => {
+    /**
+     * Total ZUSD Deposit = 100k
+     * APR 5% in 1 year
+     * ZUSD <> SOV price = 1:1
+     * Expect to return ~5k of SOV after a year
+     */
+    const APR = toBN(500);
+    const mockPrice = toBN(dec(1, 18));
+    const timeAccrual = timeValues.SECONDS_IN_ONE_YEAR;
+    const initialTotalSOVIssued = await communityIssuanceTester.totalSOVIssued()
+    const zusdDeposits = toBN(dec(100, 21))
+
+    await communityIssuanceTester.setAPR(APR);
+    await contracts.priceFeedSovryn.setPrice(contracts.zusdToken.address, zeroToken.address, mockPrice.toString());
+
+    await assert.equal(initialTotalSOVIssued, 0);
+
+    // Fast forward time
+    await th.fastForwardTime(timeAccrual, web3.currentProvider);
+
+    // Issue SOV
+    await communityIssuanceTester.unprotectedIssueSOV(zusdDeposits);
+
+    // const latestLastIssuanceTime = await communityIssuanceTester.lastIssuanceTime();
+    const latestTotalSOVIssued = await communityIssuanceTester.totalSOVIssued();
+    const expectedTotalSOVIssued = '5000000000000000000000';
+
+    assert.isAtMost(th.getDifference(latestTotalSOVIssued, expectedTotalSOVIssued), 1000000000000000);
```

Comment:
where does the difference come from?

Reply:
There is a slightly delay for the blockTimestamp between the test and when the smart contract function execution.
Here is where the difference come from